src/Dapperer/QueryBuilders/MsSql/SqlQueryBuilder.cs

```diff
@@ -89,11 +89,20 @@ public string GetAll<TEntity>()
             return string.Format("SELECT * FROM {0} ", tableInfo.TableName);
         }
 
-        public PagingSql PageQuery<TEntity>(long skip, long take, string orderByQuery = null, string filterQuery = null, ICollection<string> additionalTableColumns = null)
+        public PagingSql PageQuery<TEntity>(long skip, long take, string fromQuery = null, string orderByQuery = null, string filterQuery = null, ICollection<string> additionalTableColumns = null)
             where TEntity : class
         {
             ITableInfoBase tableInfo = GetTableInfo<TEntity>();
 
+            if(string.IsNullOrWhiteSpace(fromQuery))
+            {
+                fromQuery = tableInfo.TableName;
+            }
+            else
+            {
+                fromQuery = string.Format("({0})", fromQuery);
+            }
+
```

Comment:
line 111 references the` tableInfo.TableName` inside the method implementation
Sounds like a place where a swap with `fromQuery` would be needed

Reply:
I'm not sure we'd like to change the logic there as it is pointing to what columns to get. The `[TableName].*` sounds like the correct statement there